src/main/pages/user-management/permissions.adoc

```diff
@@ -50,135 +48,141 @@ The following permissions are applicable to organizations:
 [width="100%",cols="30%,70%",options="header",]
 |===
 |Permission |Description
-|update |Allows editing of organization settings and information.
+|update |Allows editing of the organization settings and information.
 |delete |Allows deleting an organization.
 |manageSuborganizations |Allows creating and managing sub-organizations.
-|manageResources |Allows redistribution of an organization’s resources and defining resource limits.
+|manageResources |Allows redistribution of an organization’s resources and defining the resource limits.
 |manageWorkspaces |Allows creating and managing all the organization’s workspaces.
-|setPermissions |Allows adding and removing users as well as updating their permissions.
+|setPermissions |Allows adding and removing users and updating their permissions.
 |===
 
 [id="system-permissions"]
-== System Permissions
+== System permissions
 
-System permissions control aspects that affect the whole Che installation.
+System permissions control aspects of the whole Che installation.
 
-The following permissions are applicable to organizations:
+The following permissions are applicable to the organization:
 
 [cols=",",options="header",]
 |===
 |Permission |Description
-|manageSystem |Allows control of the system, workspaces and organizations.
-|setPermissions |Allows updating of permissions for users on the system.
+|manageSystem |Allows control of the system, workspaces, and organizations.
+|setPermissions |Allows updating the permissions for users on the system.
 |manageUsers |Allows creating and managing users.
 |===
 
-[id="super-priviliged-mode"]
-== Super Priviliged Mode
-
-The permission "manageSystem" can be extended to provide a super privileged mode that allows advanced actions to be performed on any resources managed by the system. A user with "manageSystem" permission is able read and stop any workspaces. To perform other actions on workspaces and organizations, the user will need to assign himself the permissions needed.
+[id="super-privileged-mode"]
+== Super-privileged mode
 
-By default, this mode is disabled.
+The permission "manageSystem" can be extended to provide a super-privileged mode. This allows the user to perform advanced actions on any resources managed by the system. You can read and stop any workspaces with the  "manageSystem" permission and assign permissions to users as needed.
 
-It is possible to activate this option by configuring the `CHE_SYSTEM_SUPER_PRIVILEGED_MODE` in the `che.env` file.
+The super-privilege mode is disabled by default. You can change to super-privilege mode by configuring the `CHE_SYSTEM_SUPER_PRIVILEGED_MODE` variable in the `che.env` file.
```

Comment:
I'd suggest changing "super-privilege mode" to "super-privileged mode" in order to keep it consistent:

```adoc
Super-privileged mode is disabled by default. You can change to super-privileged mode by configuring the `CHE_SYSTEM_SUPER_PRIVILEGED_MODE` variable in the `che.env` file.
```

Also, what do we need to change the variable to? I'm assuming `true`. Likewise, _when_ can we change this variable? Does it work no matter when it's changed?

Reply:
Great questions. I plan to check it out for myself.